### ISSUE TITLE
Issues with 8192cu driver on 3.5.6 image for Pi

### ISSUE THREAD

**Issue by rudybrian** [FalconChristmas/fpp#634]
Hi folks,

I recently rebuilt a Pi2B that had been reliably running 1.10 with the 1.8 image for the past two years and immediately ran into issues with the wireless adapter (Edimax EW-7811Un which is a 8192cu variant). After first bootup with the 3.5.6 image, wlan0 wasn't appearing when set to use the default External drivers. I switched to Kernel drivers which discovered the device (via the rtl8192cu kernel module), but this driver is extremely unreliable and would only occasionally associate with my wireless access point. I did some digging and found /etc/modprobe.d/blacklist-8192cu.conf was preventing the 8192cu driver from loading when using External drivers. After removing this file, the /lib/modules/4.19.75-v7+/kernel/drivers/net/wireless/realtek/rtl8192cu/8192cu.ko kernel module loaded correctly. However, I did notice that this file differs than the one found in /lib/modules/4.19.75-v7+/kernel/drivers/net/wireless:

```
fpp@FPP2:/opt/wifi/rtl8192cu-fixes $ ls -la /lib/modules/4.19.75-v7+/kernel/drivers/net/wireless
total 10728
drwxr-xr-x 11 root root    4096 Dec 31 12:26 .
drwxr-xr-x 14 root root    4096 Oct 17 11:12 ..
-rw-r--r--  1 root root  727764 Oct 17 11:26 8192cu.ko
```
and
```
fpp@FPP2:/lib/modules/4.19.75-v7+/kernel/drivers/net/wireless/realtek/rtl8192cu $ ls -la
total 732
drwxr-xr-x 2 root root   4096 Oct 17 11:12 .
drwxr-xr-x 6 root root   4096 Oct 17 11:12 ..
-rw-r--r-- 1 root root 740884 Oct 11 05:31 8192cu.ko
```
This looks to be the result of the _make install_ step from FPP-Wifi-Drivers.sh not copying the patched kernel module over the old file during installation.

**Comment by rudybrian** [comment]
FYI: To see the full path of the kernel modules being used, the following is quite helpful:

`awk '{ print $1 }' /proc/modules | xargs modinfo | grep filename | awk '{ print $2 }' | sort`

**Comment by rudybrian** [comment]
#635 should address the leftover /etc/modprobe.d/blacklist-8192cu.conf file in the 3.5.6 image, but more work is needed to load the patched kernel module.

**Comment by rudybrian** [comment]
Resolved as of 4e2c20eb1b9e1b7cf043905a7a09ff92d97820e1, but some investigation is still needed as to why there are what appears to be two versions of the same kernel module on the filesystem.

**Comment by rudybrian** [comment]
Closing out as this behaves as it should in the 3.6.1 image.
